src/app/components/TableHeaderAge/index.tsx

```diff
@@ -47,15 +58,15 @@ export const TableHeaderAge: FC = () => {
           leaveDelay={0}
           placement={'top'}
         >
-          <Button variant="text" onClick={() => changeSetting('ageHeaderType', TableAgeType.DateTime)}>
+          <StyledButton variant="text" onClick={() => changeSetting('ageHeaderType', TableAgeType.DateTime)}>
             <Typography
               sx={{
                 fontWeight: 700,
               }}
             >
-              {t('common.age')}
+              {label || t('common.age')}
```

Comment:
Not sure why only Distance would need custom label? In this case it would make sense to have a Map <TableAgeType, title>? Or am I missing something?

Reply:
For now it is all I need, if this part will need another adjustment I will open PR in the future.